### PR TITLE
About us page team section

### PR DESCRIPTION
**Pull request by sara-mohammed96**
the team section contains the information a bout the each member , I used mdb for the cards and tailwind for the rest of the styling and i used animation for the cards (hover ) and also the social media  [#19 ](url)
screencapture-localhost-3000-2020-08-03-02_06_19

### CODE REVIEW

File: src/pages/AboutUs/Details.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import pic from './pic.png';
+import { MDBRow, MDBCol } from 'mdbreact';
+
+function Details() {
+  const aboutPageEnglishTranslation = {
+    headerTitle: 'About us',
+    text:
+      'It is a long established fact that a reader will be distracted bythe readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of letters, as opposed to using  content hereh. It is a long established fact that a reader will be distracted by the readable content of a page when looking at its layout. The point of using Lorem Ipsum is that it has a more-or-less normal distribution of etters, as opposed to using Content h ereh.',
+    ReadMoreText: 'Read more',
+  };
```
Comment (MohammedAlabd) [inline src/pages/AboutUs/Details.js:11]: can you move those to __i18next__ and use it here

File: src/pages/AboutUs/Team.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                <MDBIcon>
+                  <a
+                    href="#"
+                    className="fab fa-linkedin pr-2 fa-2x blue-text transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110"
+                  ></a>
+                </MDBIcon>
+              </div>
+            </MDBCardBody>
+          </MDBCard>
```
Comment (MohammedAlabd) [inline src/pages/AboutUs/Team.js:128]: from line 78 to 128

move this to a separate component and call it `TeamMemberCard.js`

File: public/locales/en/translation.json [hunk 3]
```diff
@@ -1 +1,21 @@
-{}
+{
+  "aboutPage": {
+    "saraAltaee": "Sarah Al-Taee",
+    "saraBio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",
+    "specailty": "designer",
+    "neven": "Neven Bedr",
+    "nevenBio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",
+
+    "dunia": "Dunia Nasr",
```
Comment (MohammedAlabd) [inline public/locales/en/translation.json:9]: maybe you can make it one level deeper 
puting each team in a different object 
```suggestion
"sara": { "fullName": "Sarah Al-Taee",
               "bio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",
               "specialty": "designer"
            },
"neven": {"fullName": "Neven Bedr",
                  "bio": "Nemo enim ipsam voluptatem quia voluptas sit aspernatu  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem.",

    "dunia": "Dunia Nasr",
```

File: src/pages/AboutUs/Details.js (hunk 1 above)
Comment (MohammedAlabd) [reply src/pages/AboutUs/Details.js:11]: did you moved these lines to i18next

File: src/pages/AboutUs/Team.js [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+    },
+    {
+      member: {
+        name: t('aboutPage.zainab'),
+        bio: t('aboutPage.zainabBio'),
+        specailty: t('aboutPage.specailty'),
+        pic: pic5,
+      },
+    },
+  ];
```
Comment (MohammedAlabd) [inline src/pages/AboutUs/Team.js:62]: this logic is a bit overwhelming

try to fix the i18next above and that could give you some idea about how to improve this chunk of code

File: src/pages/AboutUs/TeamMemberCards.js [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+function TeamMemberCards({ member }) {
+  return (
+    <MDBCard
+      className=" animate transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110  content-start max-w-xs m-5 bg-blue-100 "
+    >
+      <img
+        style={{ width: '9rem', marginTop: '-25%' }}
+        className=" rounded-circle img-fluid z-depth-1 mx-auto"
+        src={member.member.pic}
```
Comment (MohammedAlabd) [inline src/pages/AboutUs/TeamMemberCards.js:14]: try to find a way not to use member.member twice 
and use destructure here to make it better

File: src/pages/AboutUs/TeamMemeberCards.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
Comment (MohammedAlabd) [inline src/pages/AboutUs/TeamMemeberCards.js:1]: why did you have two files doing the same thing?

File: src/componenets/AboutUs/Team.js [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+    { name: 'neven', pic: pic1, id: 2 },
+    { name: 'dunia', pic: pic2, id: 3 },
+    { name: 'qays', pic: pic3, id: 4 },
+    { name: 'zeena', pic: pic4, id: 5 },
+    { name: 'zainab', pic: pic5, id: 6 },
+  ];
+  const teamInfo = t(`aboutPage.members`, { returnObjects: true });
+  const team = members.map((member) => {
+    const { name } = member;
+    return Object.assign({}, member, teamInfo[name]);
```
Comment (MohammedAlabd) [inline src/componenets/AboutUs/Team.js:25]: that's such an awesome one .
good job here

File: src/componenets/AboutUs/TeamMemberCards.js [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+import PropTypes from 'prop-types';
+function TeamMemberCards({ member }) {
+  const { pic, fullName, specailty, bio } = member;
+  return (
+    <MDBCard
+      className="animate transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110 content-start max-w-xs m-5 bg-blue-100 "
+    >
+      <img
+        style={{ width: '9rem', marginTop: '-25%' }}
```
Comment (MohammedAlabd) [inline src/componenets/AboutUs/TeamMemberCards.js:12]: don't use inline style and use classes instead please

File: src/componenets/AboutUs/Team.js [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+          </h1>
+          <br />
+          <div className="flex justify-center">
+            <hr className="w-48" />
+          </div>
+        </div>
+      </MDBCol>
+      <div className=" row m-10 flex justify-center mt-20">
+        {team.map((member) => (
+          <TeamMemberCards key={member.id} member={member}></TeamMemberCards>
```
Comment (QaysAyad) [inline src/componenets/AboutUs/Team.js:43]: You can do it like this 
```suggestion
          <TeamMemberCards key={member.id} member={member} />
```

File: src/componenets/AboutUs/TeamMemberCards.js [hunk 10]
```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { MDBCard, MDBCardBody, MDBIcon } from 'mdbreact';
+import PropTypes from 'prop-types';
+function TeamMemberCards({ member }) {
+  const { pic, fullName, specailty, bio } = member;
+  return (
+    <MDBCard
+      className="animate transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110 content-start max-w-xs m-5 bg-blue-100 "
```
Comment (QaysAyad) [inline src/componenets/AboutUs/TeamMemberCards.js:9]: Delete this extra space
```suggestion
                        transform hover:-translate-y-1 hover:scale-110 content-start max-w-xs m-5 bg-blue-100"
```

File: src/pages/AboutUs/Team.js [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+          </h1>
+          <br />
+          <div className="flex justify-center">
+            <hr className="w-48" />
+          </div>
+        </div>
+      </MDBCol>
+      <div className=" row m-10 flex justify-center mt-20">
+        {team.map((member) => (
+          <TeamMemberCards member={member}></TeamMemberCards>
```
Comment (QaysAyad) [inline src/pages/AboutUs/Team.js:79]: I can be written like this
```suggestion
          <TeamMemberCards member={member} />
```

File: src/pages/AboutUs/Team.js [hunk 12]
```diff
@@ -0,0 +1,87 @@
...
+        ))}
+      </div>
+      ;
```
Comment (QaysAyad) [inline src/pages/AboutUs/Team.js:82]: is this needed?

File: src/pages/AboutUs/TeamMemberCards.js [hunk 13]
```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { MDBCard, MDBCardBody, MDBIcon } from 'mdbreact';
+import PropTypes from 'prop-types';
+
+function TeamMemberCards({ member }) {
+  const { pic, name, specailty, bio } = member.member;
+  return (
+    <MDBCard
+      className=" animate transition duration-500 ease-in-out  
+                        transform hover:-translate-y-1 hover:scale-110  content-start max-w-xs m-5 bg-blue-100 "
```
Comment (QaysAyad) [inline src/pages/AboutUs/TeamMemberCards.js:10]: extra space
```suggestion
                        transform hover:-translate-y-1 hover:scale-110  content-start max-w-xs m-5 bg-blue-100"
```

File: src/pages/AboutUs/TeamMemberCards.js [hunk 14]
```diff
@@ -0,0 +1,62 @@
...
+    >
+      <img
+        style={{ width: '9rem', marginTop: '-25%' }}
+        className=" rounded-circle img-fluid z-depth-1 mx-auto"
```
Comment (QaysAyad) [inline src/pages/AboutUs/TeamMemberCards.js:14]: extra space

```suggestion
        className="rounded-circle img-fluid z-depth-1 mx-auto"
```

**Pull request merged**